portal/templates/flask_user/register.html

```diff
@@ -24,17 +24,17 @@ <h4 class="tnth-headline reg-title">{{ app_text('registration title') }}</h4>
                 {{ form.hidden_tag() }}
                 {# Username or Email #}
                 {% set field = form.username if user_manager.enable_username else form.email %}
-                {{ render_field(field, label="Email address", label_visible=false, tabindex=220, required="required", data_error="Email address is required." ) }}
+                {{ render_field(field, label="Email address", label_visible=false, tabindex=220, required="required", data_error="Email address is required.", class="input-field" ) }}
```

Comment:
Are you able to wrap the labels and messages with the translation call (`_()`)? 
Uncertain if it would work in a macro but these will be pretty important to translate later